code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -516,3 +516,73 @@
 		human_user.adjustStaminaLoss(-10, FALSE)
 		human_user.adjustToxLoss(-10, FALSE)
 		human_user.adjustOxyLoss(-10)
+
+/obj/effect/proc_holder/spell/pointed/manse_link
+	name = "Mansus Link"
+	desc = "Piercing through reality, connecting minds. This spell allows you to add people to a mansus net, allowing them to communicate with eachother"
+	school = "transmutation"
+	charge_max = 600
+	clothes_req = FALSE
+	invocation = "PI'RC' TH' M'ND"
+	invocation_type = "whisper"
+	range = 11
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "mansus_link"
+	action_background_icon_state = "bg_ecult"
+
+/obj/effect/proc_holder/spell/pointed/manse_link/can_target(atom/target, mob/user, silent)
+	if(!ishuman(target))
```

Comment:
so... you can't use this on constructs? isn't, like, the entire point of this spell to keep your construct army coordinated?

er, demon army, my bad